package.json

```diff
@@ -164,8 +164,6 @@
     ]
   },
   "cacheDirectories": [
-    "node_modules",
-    ".cache",
-    "public"
+    "node_modules"
```

Comment:
I think Heroku does this by default (cache node_modules)

do we actually want to cache `.cache` and `public` (+ then do sync with S3) - would it make it faster? ( I would imagine it is the same S3)

Reply:
I believe caching `.cache` and `public` on Heroku adds them to the slug size, and is part of why this repo was starting to push the limit and we started with S3 in the first place.